packages/shared-ux/chrome/navigation/src/ui/components/navigation_section_ui.tsx

```diff
@@ -14,16 +14,16 @@ import {
   EuiSideNavItemType,
   EuiText,
 } from '@elastic/eui';
-import type { BasePathService, NavigateToUrlFn } from '../../../../types/internal';
-import { navigationStyles as styles } from '../../../styles';
-import { useNavigation as useServices } from '../../../services';
+import type { BasePathService, NavigateToUrlFn } from '../../../types/internal';
+import { navigationStyles as styles } from '../../styles';
+import { useNavigation as useServices } from '../../services';
 import { ChromeProjectNavigationNodeEnhanced } from '../types';
 
 const navigationNodeToEuiItem = (
   item: ChromeProjectNavigationNodeEnhanced,
   { navigateToUrl, basePath }: { navigateToUrl: NavigateToUrlFn; basePath: BasePathService }
 ): EuiSideNavItemType<unknown> => {
-  const href = item.deepLink?.href;
+  const href = item.deepLink?.href ?? item.href;
```

Comment:
It looks like we should prepend the basePath to href in a single place, to solve https://github.com/elastic/kibana/issues/158376 and not repeat the code.

Maybe something like:
```
  const rootHref = item.deepLink?.href ?? item.href;
  const href = rootHref ? basePath.prepend(rootHref) : undefined;
```

Reply:
If you don't mind I will address this in a following PR. As I mentioned earlier the href are going away in the following PR so I don't think we should fix it.